### PR TITLE
update connector V2 common-options doc

### PR DESCRIPTION
**Pull request by TaoZex**
## Purpose of this pull request
The main purpose is update connector V2 common-options doc.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [X] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: docs/en/connector-v2/sink/Assert.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ A flink sink plugin which can assert illegal data by user defined rules
 |rules.field_value              | ConfigList  | no       | -             |
 |rules.field_value.rule_type    | string      | no       | -             |
 |rules.field_value.rule_value   | double      | no       | -             |
-
+| common-options                | string      | no       | -             |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Assert.md]: Type of `common-options` maybe not `string`, I think common-options have no type.

File: docs/en/connector-v2/sink/Assert.md [hunk 2]
```diff
@@ -52,6 +52,10 @@ The following rules are supported for now
 
 the value related to rule type
 
+### common options [string]
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Assert.md]: remove `[string]`

File: docs/en/connector-v2/sink/Console.md [hunk 3]
```diff
@@ -16,6 +16,12 @@ Used to send data to Console. Both support streaming and batch mode.
 
 | name | type   | required | default value |
 | --- |--------|----------|---------------|
+| common-options| string| no | -  |
+
+
+### common options [string]
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/Console.md]: same as above.

File: docs/en/connector-v2/source/Clickhouse.md [hunk 4]
```diff
@@ -26,14 +26,14 @@ Reading data from Clickhouse can also be done using JDBC
...
-| password       | string | yes      | -             |
-| common-options | string | yes      | -             |
+| name              | type   | required | default value |
+| ----------------- | ------ | -------- | ------------- |
+| host              | string | yes      | -             |
+| database          | string | yes      | -             |
+| sql               | string | yes      | -             |
+| username          | string | yes      | -             |
+| password          | string | yes      | -             |
+| result_table_name | string | no       | -             |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/source/Clickhouse.md]: replace `result_table_name` to `common-options` and this param have no type.

**Comment by EricJoy2048** [comment]
Please remove `fields` from https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/connector-v2/source/common-options.md, because not all source connector can set `fields`.

And, find the source connector can config `fields` params and then add `fields` to the connector document.

**Comment by TaoZex** [comment]
> Please remove `fields` from https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/connector-v2/source/common-options.md, because not all source connector can set `fields`.
> 
> And, find the source connector can config `fields` params and then add `fields` to the connector document.

Thanks, I will fix it.

File: docs/en/connector-v2/sink/Assert.md (hunk 1 above)
Comment (TaoZex) [reply docs/en/connector-v2/sink/Assert.md]: The common-options used by the connector V1 version is string, do we need to change it to no?

File: docs/en/connector-v2/sink/HdfsFile.md [hunk 5]
```diff
@@ -122,6 +123,9 @@ Storage mode, currently supports `overwrite`. This means we will delete the old
 If `is_enable_transaction` is `true`, Basically, we won't encounter the same file name. Because we will add the transaction id to file name.
 
 For the specific meaning of each mode, see [save-modes](https://spark.apache.org/docs/latest/sql-programming-guide.html#save-modes)
+### common options [string]
```
Comment (hailin0) [inline docs/en/connector-v2/sink/HdfsFile.md]: add empty line before

File: docs/en/connector-v2/sink/Enterprise-WeChat.md [hunk 6]
```diff
@@ -37,6 +38,9 @@ A list of userids to remind the specified members in the group (@ a member), @ a
 ### mentioned_mobile_list [array]
 
 Mobile phone number list, remind the group member corresponding to the mobile phone number (@ a member), @ all means remind everyone
+### common options [string]
```
Comment (hailin0) [inline docs/en/connector-v2/sink/Enterprise-WeChat.md]: add empty line before

File: docs/en/connector-v2/sink/Neo4j.md [hunk 7]
```diff
@@ -64,7 +65,9 @@ maximum transaction retry time(seconds). transaction fail if exceeded
 ### max_connection_timeout [long]
 The maximum amount of time to wait for a TCP connection to be established (seconds)
 
+### common options [string]
 
+Sink plugin common parameters, please refer to [Sink Common Options](common-options.md) for details
```
Comment (hailin0) [inline docs/en/connector-v2/sink/Neo4j.md]: add empty line after

File: docs/en/connector-v2/source/FakeSource.md [hunk 8]
```diff
@@ -23,6 +23,7 @@ just for testing, such as type conversion and feature testing
 | result_table_name | string | yes      | -             |
 | schema            | config | yes      | -             |
 | row.num           | long   | no       | 10            |
+| result_table_name | string | no       | -             |
```
Comment (hailin0) [inline docs/en/connector-v2/source/FakeSource.md]: as above

**Pull request closed without merging**